### PR TITLE
Fixing RichConsoleExporter to allow for multiple traces at once

### PR DESCRIPTION
**Pull request by rbizos**
List of changes:
- With a batch exporter currently, RichConsoleExporter would make all span originate with the first trace
- Also it would print any span twice which would make the output a bit difficult to read
- Additionally adding resources to the output to help debug 

This makes the debug of a SDK configuration a bit easier

### CODE REVIEW

**Review by ocelotl — COMMENTED** [review]
Please add test cases

File: exporter/opentelemetry-exporter-richconsole/tests/test_rich_exporter.py [hunk 1]
```diff
@@ -45,3 +48,34 @@ def test_span_exporter(tracer_provider, span_processor, capsys):
...
+
+
+def walk_tree(root: Tree) -> int:
+    # counts the amount of spans in a tree that contains a span
+    return sum([walk_tree(child) for child in root.children]) + int(
+        "span" in root.label
+    )
+
+
+def test_multiple_traces(tracer_provider):
```
Comment (ocelotl) [inline exporter/opentelemetry-exporter-richconsole/tests/test_rich_exporter.py:60]: Please add a test that checks that the parent-child relationship between spans is correct.

Comment (rbizos) [reply exporter/opentelemetry-exporter-richconsole/tests/test_rich_exporter.py:65]: Done :)

File: exporter/opentelemetry-exporter-richconsole/src/opentelemetry/exporter/richconsole/__init__.py [hunk 2]
```diff
@@ -73,6 +73,7 @@ def _ns_to_time(nanoseconds):
 
 
 def _child_to_tree(child: Tree, span: ReadableSpan):
+    print(span.name)
```
Comment (ocelotl) [inline exporter/opentelemetry-exporter-richconsole/src/opentelemetry/exporter/richconsole/__init__.py:76]:
```suggestion
```

**Review by ocelotl — CHANGES_REQUESTED** [review]
Please remove the `print`

File: exporter/opentelemetry-exporter-richconsole/src/opentelemetry/exporter/richconsole/__init__.py (hunk 2 above)
Comment (rbizos) [reply exporter/opentelemetry-exporter-richconsole/src/opentelemetry/exporter/richconsole/__init__.py]: Ah my bad

**Review by ocelotl — APPROVED** [review]

**Comment by ocelotl** [comment]
Please fix `lint` failures

File: CHANGELOG.md [hunk 3]
```diff
@@ -51,8 +51,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#1313](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1313))
 - Fix uninstrumentation of existing app instances in FastAPI
   ([#1258](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1258))
-- Fix uninstrumentation of existing app instances in falcon
-  ([#1341]https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1341)
```
Comment (srikanthccv) [inline CHANGELOG.md:55]: Why remove this? And also move your fix to unreleased section.

Comment (rbizos) [reply CHANGELOG.md:55]: That's an error on my side

**Comment by rbizos** [comment]
Took some back and forth, sorry about that, I did not spend enough on getting the dev environment to work properly as my feature was so simple. Had some similar issue than https://github.com/open-telemetry/opentelemetry-python-contrib/issues/1083. Now that all works it should be better.

**Pull request merged**